### ISSUE TITLE
Bangalore, November 10th

### ISSUE THREAD

**Issue by gireeshpunathil** [nodejs/code-and-learn#76]
I am planning to run a `code&learn` in Bangalore this Friday the 10th, and am expecting contribution from ~10 people. Request your support in hand-holds and reviews, thanks in advance. Some of the changes I have ear-marked are:
 - few typo corrections in docs
 - few string concatenation replacement with template literals
 - few common.mustCall wrappings

Suggestions on any other interesting refactoring suggestions are appreciated.

**Comment by fhinkel** [comment]
That's great. Thanks so much! I'll try to do a review on Saturday of the PRs.

**Comment by benjamingr** [comment]
@thefourtheye if you happen to have time :)

**Comment by fhinkel** [comment]
Thanks for running the Code and Learn @gireeshpunathil. I have time for some reviews. Is there a particular label I can search for?

**Comment by gireeshpunathil** [comment]
thanks @fhinkel . `code&learn` is the label. PRs #16910 - #16933 , almost continuous .

**Comment by gireeshpunathil** [comment]
https://github.com/nodejs/node/pull/16910 - https://github.com/nodejs/node/pull/16933

**Comment by fhinkel** [comment]
Wow, that's a lot! Great to see so many new contributors ❤️ . Thanks so much for organizing @gireeshpunathil. I was able to land a bunch of them, waiting for the CI on a few more. Thanks to @jasnell, @tniessen, @cjihrig, @danbev, and @vsemozhetbyt  for all the reviews 🙏 .

**Comment by Trott** [comment]
There seem to be a lot of changes in `lib` in these issues. I tend to avoid those for Code + Learn because it can mean people end up making changes that need to be benchmarked, and that can be a bit more than people are prepared to undertake at first.  Hopefully none of these are in hot paths and/or the perf differences of these types of changes are known to be so insignificant that it doesn't matter.

**Comment by Trott** [comment]
Overall, looks really good. Nice job, @gireeshpunathil! Some smaller Code + Learn events have not gone so well. I think @benjamingr has run some good ones too.

**Comment by gireeshpunathil** [comment]
@Trott - Acknowledged. For the `lib`, I have cautioned people on the significance and sensitivity of the module and recommended to pick up only simple, and obvious changes, if at all. I hope that has been abided by.

@fhinkel - thank you very much for the review, and especially the kind messages you left on the landing, that means a lot to the new co ers I am sure!

Thank you all the mentors who reviewed with absolute receptiveness, approachability and sympathy - I am sure this has cast the best PR experience possible to the contributors which they will cherish for a long time and drive them to come back.


Closing this issue as its purpose is met, having almost all the PRs in the codebase.
